.travis.yml

```diff
@@ -52,23 +52,22 @@ jobs:
     # This tests the formatting of a PR.
     - if: (type = pull_request) OR (type = cron) OR NOT (branch = master)
       stage: test
-      name: "pgindent and license check"
+      name: "clang-format and license check"
       env:
-        - PG_VERSION=10.2 PG_GIT_TAG=REL_10_2 ADDITIONAL_CMAKE_FLAGS="-DUSE_DEFAULT_VISIBILITY=1"
+        - PG_VERSION=11.1 ASAN_OPTIONS="detect_leaks=0"
```

Comment:
Why do we need address sanitizer options in a format and license check?
Do we actually need the postgres version here?
I guess we do for the export prefix check which isn't mentioned in the name